### PR TITLE
add CreateRuntimeEnv Handler

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

File: pkg/manager/runtime_env/handler.go [hunk 1]
```diff
@@ -12,10 +12,41 @@ import (
...
 	"openpitrix.io/openpitrix/pkg/pb"
+	"openpitrix.io/openpitrix/pkg/utils/sender"
+	"openpitrix.io/openpitrix/pkg/logger"
 )
 
 func (p *Server) CreateRuntimeEnv(ctx context.Context, req *pb.CreateRuntimeEnvRequest) (*pb.CreateRuntimeEnvResponse, error) {
-	return nil, status.Errorf(codes.Internal, "CreateRuntimeEnv: %+v", fmt.Errorf("hello world"))
+	//TODO: validation request
+	s := sender.GetSenderFromContext(ctx)
+	logger.Infof("Got sender: %+v", s)
```
Comment (zheng1) [inline pkg/manager/runtime_env/handler.go]: These two lines can be deleted, and then we will add the general grpc debug log.

File: pkg/manager/runtime_env/bll.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package runtime_env
```
Comment (chilianyi) [inline pkg/manager/runtime_env/bll.go]: bll means Business Logic Layer? The file name may not very good.

File: pkg/manager/runtime_env/handler.go [hunk 3]
```diff
@@ -12,10 +12,41 @@ import (
...
+		req.GetDescription().GetValue(),
+		req.GetRuntimeEnvUrl().GetValue(),
+		s.UserId)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "CreateRuntimeEnv: %+v", err)
+	}
+
+	//create labels
+	err = p.createRuntimeEnvLabels(runtimeEnvId, req.Labels.GetValue())
+	if err != nil {
```
Comment (chilianyi) [inline pkg/manager/runtime_env/handler.go]: May need logger.Errorf, so back end is able to locate the problem.

File: pkg/manager/runtime_env/handler.go [hunk 4]
```diff
@@ -12,10 +12,41 @@ import (
...
+		return nil, status.Errorf(codes.Internal, "CreateRuntimeEnv: %+v", err)
+	}
+
+	//get response
+	pbRuntimeEnv, err := p.getRuntimeEnvPbWithLabel(runtimeEnvId)
+	if err != nil {
```
Comment (chilianyi) [inline pkg/manager/runtime_env/handler.go]: Ditto

File: pkg/manager/runtime_env/bll.go (hunk 2 above)
Comment (runzexia) [reply pkg/manager/runtime_env/bll.go]: Yes. Do you have any suggestions for file naming?

Comment (chilianyi) [reply pkg/manager/runtime_env/bll.go]: Sorry, I do not have a better one, can these code be merged into handler.go?

**Comment by runzexia** [comment]
done.

**Pull request closed without merging**